cypress/fixtures/listWorkflowsFlow.json

```diff
@@ -1 +1,195 @@
\ No newline at end of file
+{
+    "blocks": [
+        {
+            "type": "init",
+            "blockComment": "Mocked"
+        },
+        {
+            "type": "variable-get",
+            "name": "filterTag"
+        },
+        {
+            "type": "mapping",
+            "mapping": "(type(data)=='string' && data!='--none--') && data"
+        },
+        {
+            "type": "template",
+            "template": "{{#if data}}Filtering by tag: <strong>{{data}}</strong>{{/if}}"
+        },
+        {
+            "type": "context-save",
+            "contextKey": "filterTag"
+        },
+        {
+            "type": "mapping",
+            "mapping": "{ \"term\":\" \", \"displayHidden\":false}"
+        },
+        {
+            "type": "form",
+            "hasSubmit": true,
```

Comment:
Added the submit button to avoid state bug.

Reply:
Again, a related issue was made for this:
https://github.com/kendraio/kendraio-app/issues/429